src/shared/i18n/extension/common/fr.ts

```diff
@@ -0,0 +1,18 @@
+import { RecursivePartial } from "@/shared/types/RecursivePartial";
+import { CommonTranslations } from "./type";
+
+export const fr: RecursivePartial<CommonTranslations> = {
+    averagePerDay: '⌀ par jour',
+    chart: 'Graphique',
+    table: 'Tableau',
+    sum: 'Somme',
+    total: 'Total',
+    resourceUnits: 'Unités des ressources',
+    dsu: 'USD',
+    msu: 'USM',
+    date: 'Date',
+    dateTime: 'Date et heure',
+
+    madeWithLove1: 'Créé avec',
+    madeWithLove2: 'par Wonkydonky',
+
```

Comment:
missing closing brace
```suggestion
};
```